### DISCUSSION TITLE
Adding a .txt file to root directory of Nextra project

### DISCUSSION THREAD

**Discussion by RecursiveFun** [shuding/nextra#2827] [Q&A] [answered]
Could someone point me in the right direction on how I can add my google ads.txt file to my Nextra project where it can be accessed from the relative URL /ads.txt Everything I've tried doesn't work.

**Comment by dimaMachina** [comment, marked as answer]
Add it inside /public dir

**Comment by RecursiveFun** [comment]
My mistake. I thought that was the first thing I tried. I was accidentally adding it into another subdirectory in public. Now it works thanks!
